howToTest.md

```diff
@@ -0,0 +1,76 @@
+[Jest](https://jestjs.io/docs/en/expect) main testing framework
+[Enzyme](https://enzymejs.github.io/enzyme) - Helper testing framework for getting elements from the DOM
+
+### Basic settings:
+
+1. Create '**tests**' folder in component's folder
+2. Create <my-component>.spec.tsx
```

Comment:
`spec` is the nice word, but it has generally more meaning in the world of software engineering and it is less obvious than just `test`. 
I suggest to make it `<my-component>.test.tsx`, what do you think?

Reply:
It basically depends on whether we are following BDD or TDD to write our tests. [Check this article](https://www.bignerdranch.com/blog/why-do-javascript-test-frameworks-use-describe-and-beforeeach/)
You need to decide on the naming `spec` or `test` after you decide which testing approach will we adopt (TDD or BDD)